modules/govuk/manifests/apps/email_alert_api/checks.pp

```diff
@@ -48,6 +48,17 @@
     notes_url => monitoring_docs_url(email-alert-api-unprocessed-subscription-contents),
   }
 
+    @@icinga::check::graphite { 'email-alert-api-warning-content-changes':
+    ensure    => $ensure,
+    host_name => $::fqdn,
+    target    => 'transformNull(stats.gauges.govuk.email-alert-api.content_changes.warning_total)',
+    warning   => '0',
+    critical  => '100000000',
```

Comment:
Is this value so large because this is always a warning alert and the critical should never trigger from here? Is there a better way?

Reply:
That is correct, there are existing comments in the file to this effect
```
  # We are only interested in the `warning` state but `critical` is also required
  # for valid Icinga check configuration. Setting it to a high value that won't be
  # reached allows us to get round this issue
```
I don't think we are currently aware of a better way, but are very open to ideas!

Reply:
I have no ideas. It was an observation!

Reply:
@cbaines suggested we could make the `critical` and `warning` arguments optional in the Icinga check template configuration(I think!?) which means we wouldn't need to do this. Having seen the other two healthchecks we need to set up in a similar way maybe it's worth doing this?

Reply:
I think that's still a possibility, but not quite worth doing yet. I think it would be good to see the first iteration of the extracted healthchecks in action for a bit, and have some data to work with, and then we'll be able to make more informed decisions.